### DISCUSSION TITLE
what the name of the data folder, which is to be volumned in docker ?

### DISCUSSION THREAD

**Discussion by corpusly** [apache/kvrocks#350] [General]
docker run -itd  -v /mydata:??? -p 6666:6666 kvrocks/kvrocks

**Comment by git-hulk** [comment]
the default data was `/tmp/kvrocks` in Docker

**Reply by ShooterIT** [reply]
Users also could change the data directory in `kvrocks.conf`, right?

**Reply by git-hulk** [reply]
No actually, the user mainly doesn't want to lose the data in docker after restarting. Has sent the PR #392  to resolve this issue.
